state-synchronizer/src/executor_proxy.rs

```diff
@@ -102,15 +93,14 @@ impl ExecutorProxy {
 
     // TODO make this into more general trait method in `on_chain_config`
     // once `StorageRead` trait is replaced with `DbReader` and `batch_fetch_config` method is no longer async
-    async fn fetch_all_configs(client: Arc<dyn StorageRead>) -> Result<OnChainConfigPayload> {
+    fn fetch_all_configs(storage: &dyn DbReader) -> Result<OnChainConfigPayload> {
```

Comment:
rebase and it should be a instance method now.

Reply:
my bad, it took `self` on my old repo before rebase. (although I didn't get why the change.)